pybossa/core.py

```diff
@@ -118,6 +122,27 @@ def configure_app(app):
     app.url_map.strict_slashes = app.config.get('STRICT_SLASHES')
 
 
+def setup_request_profiler(app):
+    @app.before_request
+    def start_yappi():
+        if request.args.get('profileit') == '1':
+            yappi.set_clock_type('wall')
+            yappi.start()
+            g.yappiing = True
```

Comment:
is g a flask object and doesn't need to import?

Reply:
thank you, I forgot to remove this